addons/uis/apps/ios.md

```diff
@@ -11,11 +11,48 @@ title: iOS openHAB App
 
 We provide a native iOS app for openHAB, compatible with iPhone and iPad.
 It uses the REST API of openHAB to render sitemaps of your openHAB installation.
-It also supports [myopenhab.org](http://www.myopenhab.org) including push notifications.
+
+**Features:**
+
+* View and control openHAB sitemaps, similar to the BasicUI
+* Connect to local openHAB instances on the local network
+* Connect to your openHAB instance remotely using myopenHAB
```

Comment:
Please modify this point or add another making clear that every accessible remote openHAB instance can be used.
Also (the Android offers this option) the app first tries the local, then the external address - that should be mentioned as a feature